.github/workflows/release.yml

```diff
@@ -746,9 +746,33 @@ jobs:
       with:
         arch: ${{ matrix.host }}
 
+    - name: Save code signing certificate to disk
+      id: save_cert
+      uses: timheuer/base64-to-file@v1
```

Comment:
Any chance we can prevent this? I really do not like that we store it on disk in another step then it being used, especially, as it is not removed afterwards.

Maybe a solution is, as Powershell can base64 decode, doing it in the same step as `Import code signing certificate`, and removing the (now temporary) file after the import?

Reply:
Unfortunately there is no PS native way to decode base64, but maybe this is helpful, assuming the certificate is mounted as environment variable
```Powershell
$bytes = [System.Convert]::FromBase64String("$(cat env:/WIN_CERTIFICATE_P12)")
[IO.File]::WriteAllBytes("$(pwd)\temp.p12", $bytes)
# import it 
rm "$(pwd)\temp.p12"
```

You would think there are some nice .NET methods to import a cert directly from a byte array, but these methods are not nice at all ;-)

Reply:
I wasn't sure if multi-line environment variables were even a thing on Windows to be honest, so didn't try it that way, but can certainly see if it works. Thanks for the sample.